routes/web.php

```diff
@@ -28,4 +28,8 @@
     return Inertia::render('Dashboard');
 })->middleware(['auth', 'verified'])->name('dashboard');
 
-require __DIR__.'/auth.php';
+Route::post('/save', \App\Http\Controllers\SaveGameController::class)->name('game.save');
+Route::get('/game', \App\Http\Controllers\ShowGameController::class);
+Route::get('/load', \App\Http\Controllers\LoadGameController::class)->name('game.load');
```

Comment:
```suggestion
Route::prefix('/courses')->name('courses')->group(function() {
    Route::prefix('/{course}/lessons')->name('.lessons')->group(function() {
        Route::get('{lesson}', \App\Http\Controllers\Lessons\ShowLessonController::class)->name('.show');
        Route::post('{lesson}/save', \App\Http\Controllers\Lessons\SaveLessonController::class)->name('.save');
        Route::get('{lesson}/load', \App\Http\Controllers\Lessons\LoadLessonController::class)->name('.load');
    });
});
```

We want to try to structure this towards courses and lessons so I think we should refactor this to the suggestion above. 

Having the course model and the lesson model in the route will also help you when saving the state and loading the lesson. In addition, Laravel is smart enough to resolve the models from the route saving you from having to query the database to find the correct models.